src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -137,40 +157,65 @@
 	
 	
 	
-	//Misc Weapons
+	//Dice
 	public static final RegistryObject<Item> FLUORITE_OCTET = REGISTER.register("fluorite_octet", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 4, -3.0F).efficiency(1.0F).add(OnHitEffect.RANDOM_DAMAGE), new Item.Properties().tab(MSItemGroup.WEAPONS).defaultDurability(4096).rarity(Rarity.EPIC)));
 	
+	
+	//Claws
+	public static final RegistryObject<Item> MAKESHIFT_CLAWS_DRAWN = REGISTER.register("makeshift_claws_drawn", () -> new WeaponItem(new WeaponItem.Builder(Tiers.STONE, 2, -1.5F).efficiency(10.0F).set(MSItemTypes.CLAWS_TOOL).set(ItemRightClickEffect.switchTo(MSItems.MAKESHIFT_CLAWS_SHEATHED)), new Item.Properties().defaultDurability(200).tab(MSItemGroup.WEAPONS)));
```

Comment:
00ps, just figured 0ut i can reply here
fr0m what i can remember when i used the spreadsheet it said the DPS was far l0wer than that i think? it might be an issue with the sheet but regardless it is n0t intenti0nal f0r a pre-entry st0ne tier weap0n s0 it will be changed!